### ISSUE TITLE
The result from open_llm_leaderboard is not as expected.

### ISSUE THREAD

**Issue by chi2liu** [openlm-research/open_llama#48]
open_llm_leaderboard had updated the result for open-llama-3b and open-llama-7b.

![image](https://github.com/openlm-research/open_llama/assets/19861628/86042387-0d3f-4444-81f4-bc9071d0d66c)

This result is much worse than llama-7b and does not match expectations. Is it because of the fast tokenizer issue mention in the document?

**Comment by gjmulder** [comment]
Relative scores compared to llama-7b:

![image](https://github.com/openlm-research/open_llama/assets/10833500/4c781284-a535-4741-a31d-029b560d0996)

There's a clear performance hit for the multi-shot tasks, as compared to llama-7b

**Comment by codesoap** [comment]
@c0bra There has not yet been a new release of huggingface/transformers since the fix has been merged: https://github.com/huggingface/transformers/releases. I assume we still need to wait for this.

The already existing entries for OpenLLaMa on the leader-board disappeared around a week ago as well. Maybe there is a connection and the maintainers of the leader-board removed the results, because they learned of the bug and are now waiting for the next release of huggingface/transformers... That's just my guess, though.

**Comment by young-geng** [comment]
@codesoap Yeah I've contacted the maintainers for the leaderboard for a re-evaluation request, and the model should be in the queue right now.

**Comment by gjmulder** [comment]
[open-llama-7b-open-instruct](https://huggingface.co/VMware/open-llama-7b-open-instruct) is pending evaluation in [open_llm_leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard). They confirmed that they fine-tuned with `use_fast = False`

**Comment by HeegyuKim** [comment]
OpenLLaMa 3B result is not pending. is there any reason?
